Python/examples/example.py

```diff
@@ -38,10 +38,18 @@
 #     trees=500,
 #     numCores=cpu_count(),
 # )
+#
```

Comment:
instead of modifying this example.py (which isn't very easy to discover) -- I think we should create a new notebook showing how s-rerf does vs. sklearn/ternary/binary for mnist possibly. which can go on the docs page. :)

Reply:
one more thing -- can you add a quick test to show that s-rerf is actually callable from Python?

just a quick addition to `test_RerF.py`

Reply:
Done. https://github.com/neurodata/RerF/blob/c6febfee1dd10e4e8e94698e3b2a21e3290446b7/Python/tests/test_RerF.py#L169